x-pack/plugins/alerting/server/alert/alert.ts

```diff
@@ -79,9 +87,12 @@ export class Alert<
         this.scheduledExecutionOptions
       )
     ) {
-      if (actionHash) {
+      if (uuid && actionHash) {
```

Comment:
could there ever be a case where we have the hash and not a UUID?

Reply:
Since `actionInState` will take either, seems like we could do `uuid || actionHash` here.

Reply:
We used to pass only actionHash to update lastScheduledAction per action per alert. So it means this is a throttled action. Now, as we start using uuid we always pass both of them, and actionHash is supposed to be removed later...

But as the real identifier is uuid, i removed actionHash form the if condition and added in the actionHash remove part where it is required.